src/Entities/Player.cpp

```diff
@@ -2573,8 +2574,8 @@ void cPlayer::UpdateMovementStats(const Vector3d & a_DeltaPos, bool a_PreviousIs
 			// If a jump just started, process food exhaustion:
 			if ((a_DeltaPos.y > 0.0) && a_PreviousIsOnGround)
 			{
-				m_Stats.AddValue(statJumps, 1);
-				AddFoodExhaustion((IsSprinting() ? 0.008 : 0.002) * static_cast<double>(Value));
+					m_Stats.AddValue(statJumps, 1);
```

Comment:
This is an erroneous change.

Reply:
forgot to restore the tabs, first i've added a PlayerIsJumping hook there, later on i've removed, but forgot the tabulation